src/main/java/org/gitlab4j/api/models/MergeRequest.java

```diff
@@ -281,6 +286,14 @@ public void setMergeStatus(String mergeStatus) {
         this.mergeStatus = mergeStatus;
```

Comment:
Shouldn't be this setter annotated as well?

Reply:
Maybe yes, maybe no. Technically it'd certainly be correct I can't imagine a use case where someone's integration code is going to call the _setter_. You can't actually modify the merge status on GitLab.